src/AI_ExtensibleSaveFormat/AI.ExtendedSave.Hooks.cs

```diff
@@ -14,11 +22,248 @@ internal static partial class Hooks
             [HarmonyPatch(typeof(CustomClothesFileInfoAssist), nameof(CustomClothesFileInfoAssist.CreateClothesFileInfoList))]
             [HarmonyPatch(typeof(GameCoordinateFileInfoAssist), nameof(GameCoordinateFileInfoAssist.CreateCoordinateFileInfoList))]
             private static void CreateListPrefix() => LoadEventsEnabled = false;
+
             [HarmonyPostfix]
             [HarmonyPatch(typeof(CustomCharaFileInfoAssist), nameof(CustomCharaFileInfoAssist.CreateCharaFileInfoList))]
             [HarmonyPatch(typeof(CustomClothesFileInfoAssist), nameof(CustomClothesFileInfoAssist.CreateClothesFileInfoList))]
             [HarmonyPatch(typeof(GameCoordinateFileInfoAssist), nameof(GameCoordinateFileInfoAssist.CreateCoordinateFileInfoList))]
             private static void CreateListPostfix() => LoadEventsEnabled = true;
+
+
+            public static void InstallMainGameHooks(Harmony harmony)
+            {
+                // Minimizing the interference between games to limiting the harmony hook initialization
+                // MainGame Save/Load - PostFix
+                harmony.Patch(
+                    typeof(SaveData).GetMethod("Load", new[] {typeof(BinaryReader)}),
+                    null, new HarmonyMethod(typeof(Hooks).GetMethod(nameof(OnLoadMainGame)))
+                );
+                harmony.Patch(
+                    typeof(SaveData).GetMethod("SaveFile", new[] {typeof(BinaryWriter)}),
+                    null, new HarmonyMethod(typeof(Hooks).GetMethod(nameof(OnSaveMainGame)))
+                );
+
+                // HousingData Save/Load - Transpiler
+                // Based on 2021 Version
```

Comment:
Does it work on the Steam version and Japanese versions both with and without DX?

Reply:
Well currently I don't have multiple versions of AI atm.  I need to figure out how to obtain old versions first